src/objects/RecurringDonationChangeHistory__c.object

```diff
@@ -0,0 +1,389 @@
+        <inlineHelpText>The previous payment schedule for the related Recurring Donation.</inlineHelpText>
+        <label>Previous Schedule</label>
+        <length>255</length>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Previous_Status_Reason__c</fullName>
+        <description>The previous Status Reason for the related Recurring Donation.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The previous Status Reason for the related Recurring Donation.</inlineHelpText>
+        <label>Previous Status Reason</label>
+        <length>255</length>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Previous_Status__c</fullName>
+        <description>The previous Status for the related Recurring Donation.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The previous Status for the related Recurring Donation.</inlineHelpText>
+        <label>Previous Status</label>
+        <length>255</length>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Previous_Yearly_Value__c</fullName>
+        <description>The previous yearly value for this Recurring Donation.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The previous yearly value for this Recurring Donation.</inlineHelpText>
+        <label>Previous Yearly Value</label>
+        <precision>18</precision>
+        <required>false</required>
+        <scale>2</scale>
+        <trackTrending>false</trackTrending>
+        <type>Currency</type>
+    </fields>
+    <fields>
+        <fullName>Recurring_Donation__c</fullName>
+        <description>The Recurring Donation related to this Change History record.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The Recurring Donation related to this Change History record.</inlineHelpText>
+        <label>Recurring Donation</label>
+        <referenceTo>npe03__Recurring_Donation__c</referenceTo>
+        <relationshipLabel>Recurring Donation Change History</relationshipLabel>
+        <relationshipName>RD_Change_History</relationshipName>
+        <relationshipOrder>0</relationshipOrder>
+        <reparentableMasterDetail>false</reparentableMasterDetail>
+        <trackTrending>false</trackTrending>
+        <type>MasterDetail</type>
+        <writeRequiresMasterRead>false</writeRequiresMasterRead>
+    </fields>
+    <label>Recurring Donation Change History</label>
+    <nameField>
+        <displayFormat>RDCH-{0000000}</displayFormat>
+        <label>Recurring Donation Change History ID</label>
```

Comment:
I generally think we should avoid the word "Id" for auto number fields since there's already an Id on the object.
```suggestion
        <label>Recurring Donation Change History Num</label>
```

Reply:
There is no consistency with our existing auto number fields, so I went with one that made sense. Great callout though, having 2 "Id" fields is confusing.